docs/src/content/docs/book/compile.mdx

```diff
@@ -18,7 +18,57 @@ The Tact compiler can produce various outputs, ranging from the [BoC](/book/cell
 
 With the proper [configuration settings](/book/config), you can customize the behavior of the compiler to skip the generation of some or all the [build artifacts](#artifacts), and even control the addition or exclusion of [getters for supported interfaces](/book/contracts#interfaces).
 
-Since the Tact compiler is a command-line program, some of the configuration settings can be set directly. When inside a folder with a Tact-based project, such as one created using the [Blueprint](https://github.com/ton-org/blueprint) or from the [tact-template](https://github.com/tact-lang/tact-template), refer to the `npx tact --help{:shell}` command for further instructions.
+Since the Tact compiler is a command-line program, some of the configuration settings can be set directly. When inside a folder with a Tact-based project, such as one created using the [Blueprint][bp] or from the [tact-template](https://github.com/tact-lang/tact-template), refer to the `npx tact --help{:shell}` command for further instructions.
+
+## Compiler upgrades {#upgrades}
+
+Work on the compiler is active and extensive, so be sure to check out [NPM](https://www.npmjs.com/package/@tact-lang/compiler) whenever we push new releases. To follow updates, see the [Tact Kitchen in Telegram](https://t.me/tact_kitchen) for the short scoop, or our [GitHub organization](https://github.com/tact-lang) for the whole picture in all its details.
+
+To upgrade the Tact compiler to its latest version in a [Blueprint][bp]-based project or a project created from the [tact-template](https://github.com/tact-lang/tact-template), run:
+
+<Tabs>
+ <TabItem label="yarn" icon="seti:yarn">
+   ```shell
+   # recommended
+   yarn upgrade @tact-lang/compiler
+   ```
+ </TabItem>
+ <TabItem label="npm" icon="seti:npm">
+   ```shell
+   npm i @tact-lang/compiler@latest
+   ```
+ </TabItem>
+ <TabItem label="pnpm" icon="pnpm">
+   ```shell
+   pnpm upgrade @tact-lang/compiler
+   ```
+ </TabItem>
+ <TabItem label="bun" icon="bun">
+   ```shell
+   bun update --latest @tact-lang/compiler
+   ```
+ </TabItem>
+</Tabs>
+
+Sometimes, bug fixes require us to change the existing behavior, introducing small breaking changes. To be prepared for these, make sure to read the [CHANGELOG.md](https://github.com/tact-lang/tact/blob/main/dev-docs/CHANGELOG.md) and act accordingly. Except for bug fixes, Tact tries its best not to introduce breaking changes in minor or patch versions according to [semantic versioning](https://semver.org/#summary).
+
+If you want to pin down a specific version of the compiler, simply specify it in your [project.json](https://docs.npmjs.com/cli/v10/configuring-npm/package-json) as follows: `"@tact-lang/compiler": "X.Y.Z"{:json}`, where `X.Y.Z` is the version that you want to set.
```

Comment:
Is it really enough to edit `package.json`? And next time I build my Tact project it will use a different version of Tact?